docs/apis-tools/migration-manuals/migrate-to-camunda-api.md

```diff
@@ -0,0 +1,113 @@
+---
+id: migrate-to-camunda-api
+title: Migrate to the Camunda 8 API
+description: "Migrate from Camunda's V1 component REST APIs to the V2 Camunda 8 REST API to interact with Camunda 8 clusters, activate jobs, and run user task state operations."
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+This document offers a comprehensive guide to migrate from Camunda's V1 component REST APIs (the Tasklist REST API, for example) to the V2 [Camunda 8 REST API](/apis-tools/camunda-api-rest/camunda-api-rest-overview.md).
+
+Camunda is streamlining the developer experience by creating a unified REST API for Zeebe, Operate, Tasklist, and the Identity components with endpoint parity. This will be a single REST API for the orchestration cluster for a consistent and intuitive API experience to help your teams develop process automation solutions faster.
+
+:::note
+The Administration and Web Modeler APIs will not be part of the Camunda 8 REST API, as these are platform APIs outside the cluster’s scope.
```

Comment:
We'll need to refine what "cluster" means, but we have not settled on clear definitions. See https://github.com/camunda/camunda-docs/pull/4648.